Client/Frontend/Widgets/Menu/MenuViewController.swift

```diff
@@ -186,6 +186,20 @@ extension MenuViewController: MenuItemDelegate {
         performMenuAction(menuItem.action, withAnimation: animation, onView: menuItemCell.menuImageView)
     }
 
+    func menuView(menuView: MenuView, didLongPressItemAtIndexPath indexPath: NSIndexPath) {
```

Comment:
There is a lot of duplicated code here between `didLongPressItemAtIndexPath` and `didSelectItemAtIndexPath` around the highlighting of images etc. Can we maybe factor out the shared code into a separate function?